coralproject/talk
# dont start scheduledJobs if $DISABLE_JOB_PROCESSORS=true

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-1692 by declining to start scheduled jobs if the env var $DISABLE_JOB_PROCESSORS is set

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
Set `export DISABLE_JOB_PROCESSORS=true`, start the app, and observe that no scheduled jobs are run.
 
## How do we deploy this PR?
No special considerations should be needed.
